plugins/inline-row-separators/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@blockly/inline-row-separators",
```

Comment:
Based on https://developers.google.com/blockly/guides/contribute/samples/naming I think we probably want this to be `renderer-inline-row-separators`. But happy to receive push back on this.

Reply:
Done!